test/extensions/transport_sockets/tls/context_impl_test.cc

```diff
@@ -385,6 +385,24 @@ TEST_F(SslServerContextImplTicketTest, TicketKeySuccess) {
   EXPECT_NO_THROW(loadConfigYaml(yaml));
 }
 
+TEST_F(SslServerContextImplTicketTest, TicketKeySuccessWithCustomLifetimeHint) {
+  // Both keys are valid; no error should be thrown. non default ticket's lifetime hint
+  const std::string yaml = R"EOF(
+  common_tls_context:
+    tls_certificates:
+      certificate_chain:
+        filename: "{{ test_tmpdir }}/unittestcert.pem"
+      private_key:
+        filename: "{{ test_tmpdir }}/unittestkey.pem"
+  session_ticket_keys:
+    keys:
+      filename: "{{ test_rundir }}/test/extensions/transport_sockets/tls/test_data/ticket_key_a"
+      filename: "{{ test_rundir }}/test/extensions/transport_sockets/tls/test_data/ticket_key_b"
+  session_timeout: 2307s
+)EOF";
+  EXPECT_NO_THROW(loadConfigYaml(yaml));
```

Comment:
I don't think this is a super useful test, so unless you can actually test the session_timeout I would probably just kill this test entirely since you cover this in ssl_socket_test.cc